gcloud/streaming/transfer.py

```diff
@@ -35,51 +34,99 @@
 RESUMABLE_UPLOAD = 'resumable'
 
 
+_DEFAULT_CHUNKSIZE = 1 << 20
+
+
 class _Transfer(object):
+    """Generic bits common to Uploads and Downloads.
+
+    :type stream: file-like object
+    :param stream: stream to/from which data is downloaded/uploaded.
+
+    :type close_stream: boolean
+    :param close_stream: should this instance close the stream when deleted
+
+    :type chunksize: integer
+    :param chunksize: the size of chunks used to download/upload a file.
 
-    """Generic bits common to Uploads and Downloads."""
+    :type auto_transfer: boolean
+    :param auto_transfer: should this instance automatically begin transfering
+                          data when initialized
 
-    def __init__(self, stream, close_stream=False, chunksize=None,
-                 auto_transfer=True, http=None, num_retries=5):
+    :type http: :class:`httplib2.Http` (or workalike)
+    :param http: Http instance used to perform requests.
+
+    :type num_retries: integer
+    :param num_retries: how many retries should the transfer attempt
+    """
+    def __init__(self, stream, close_stream=False,
+                 chunksize=_DEFAULT_CHUNKSIZE, auto_transfer=True,
+                 http=None, num_retries=5):
         self.__bytes_http = None
```

Comment:
I stripped the ones which didn't have `@property` accessors in order to allow test assertions w/o mangling.  I will strip the others now.